tools/run_tests/performance/scenario_config.py

```diff
@@ -118,6 +118,7 @@ def _ping_pong_scenario(name,
                         unconstrained_client=None,
                         client_language=None,
                         server_language=None,
+                        async_client_threads=0,
```

Comment:
are we forced to use the `threads` wording? If not, don't you think that would be more appropriate to make usage of a word like `processes`?

Reply:
Added `client_processes` and `server_processes`.

I will try to push through two new fields in `control.proto` where defines all these configs.